META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
What is this new file?

Reply:
This was generated from the making of the sep jar file 😅, should have git ignored it mb